### PR TITLE
Compile D2 production builds with -release

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
Blocked on https://github.com/sociomantic-tsunami/dhtnode/pull/97.

File: Build.mak [hunk 1]
```diff
@@ -2,6 +2,10 @@ export ASSERT_ON_STOMPING_PREVENTION=1
 
 ifneq ($(DVER),1)
     DC := dmd-transitional
+
+	ifeq ($F, production)
+	override DFLAGS += -release
```
Comment (stefan-koch-sociomantic) [inline Build.mak]: maybe add a tab here ?

**Comment by gavin-norman-sociomantic** [comment]
Rebased.

File: Build.mak [hunk 2]
```diff
@@ -2,6 +2,10 @@ export ASSERT_ON_STOMPING_PREVENTION=1
 
 ifneq ($(DVER),1)
     DC := dmd-transitional
+
+	ifeq ($F, production)
+		override DFLAGS += -release
+	endif
```
Comment (nemanja-boric-sociomantic) [inline Build.mak]: You're mixing tabs with spaces here.

Comment (gavin-norman-sociomantic) [reply Build.mak]: :scream:

Comment (gavin-norman-sociomantic) [reply Build.mak]: Really? Where?

Comment (nemanja-boric-sociomantic) [reply Build.mak]: All other lines have spaces, only these few lines are tabs.

Comment (nemanja-boric-sociomantic) [reply Build.mak]: Sorry, I looked closer, this is the line with spaces:

```
    DC := dmd-transitional
```

could you fix it in another commit?

Comment (gavin-norman-sociomantic) [reply Build.mak]: Updated.

**Pull request merged**